src/components/Forecast.tsx

```diff
@@ -10,42 +10,48 @@ interface Props {
 	units?: Unit
 }
 
+function useTemperatureConverter() {
+	return function(temp: number, toUnit:Unit):number{
+		switch (toUnit) {
+		      case Unit.Imperial: 
+		      	 temp=Math.floor((temp * 9/5)+32);
+				 break;
+		      default: 
+		      	 temp=Math.floor(temp + 273.15);
+				 break;
+		}
+		return temp;
+	}
+  
+}
```

Comment:
Nice but this hook has to be inside a folder named `hooks` inside `src` directory. So the path should be `src/hooks/useTemperatureConverter` and make an import in this component.

Also, the function has to receive the unit from the temperature received and be able to convert to the metric aswell. So it can convert from Imperial to Metric and Standard, from Metric to Imperial and Standard and from Standard to Metric and Imperial.